api/src/services/cache.js

```diff
@@ -0,0 +1,10 @@
+const NodeCache = require('node-cache');
+
+const opts = {
+  stdTTL: 5 * 60, // standard ttl in seconds for every generated cache element: 5 minutes
+  checkperiod: 5 * 60, // period in seconds used for the automatic delete check interval: 5 minutes
+  useClones: false
+};
+const cache = new NodeCache(opts);
+
+module.exports = cache;
```

Comment:
I'm not sure if this should be a service in its current form, because it always returns the same cache instance. If we decide to reuse it for something else it would probably be better to have a different cache so we don't accidentally collide on keys.

I guess you did it because it makes unit testing easier? If so, I'd change it so it exported a function which creates and returns new cache which then gives you a function to mock.

Reply:
Switched it to return a named cache instance.